tools/workflow/clean_aws_resources.go

```diff
@@ -66,10 +74,14 @@ func terminateEc2Instances() {
 		describeInstancesInput := ec2.DescribeInstancesInput{Filters: []*ec2.Filter{&instanceStateFilter, &instanceTagFilter}, NextToken: nextToken}
 		describeInstancesOutput, err := ec2client.DescribeInstances(&describeInstancesInput)
 
+
+
 		if err != nil {
 			log.Fatalf("Failed to get instance for error %v", err)
 		}
 
+		fmt.Print(describeInstancesOutput)
```

Comment:
Use log.

Reply:
Forgot to delete when testing yesterday. Deleted it in the newest commit.